src/Native/LDANative/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+project (LDANative)
+
+set(SOURCES
+    alias_multinomial_rng_int.cpp
+    data_block.cpp
+    hybrid_alias_map.cpp
+    hybrid_map.cpp
+    lda_document.cpp
+    lda_engine.cpp
+    lda_engine_export.cpp
+    light_doc_sampler.cpp
+    light_hash_map.cpp
+    model_block.cpp
+    utils.cpp
+)
+
+if(WIN32)
+else()
+    set_property(SOURCE segment.cpp APPEND_STRING PROPERTY COMPILE_FLAGS " -msse4.1")
```

Comment:
>segment.cpp [](start = 24, length = 11)

you don't have to copypaste this line from FastTree cmake :)

Reply:
#Resolved

---
In reply to: [196948917](https://github.com/dotnet/machinelearning/pull/377#discussion_r196948917) [](ancestors = 196948917)